doc/classes/ProjectSettings.xml

```diff
@@ -811,6 +814,12 @@
 		<member name="internationalization/locale/test" type="String" setter="" getter="" default="&quot;&quot;">
 			If non-empty, this locale will be used when running the project from the editor.
 		</member>
+		<member name="internationalization/locale/translation_remaps" type="PackedStringArray" setter="" getter="">
+			Locale-dependent resource remaps. Edit them in the "Localization" tab of Project Settings editor. 
```

Comment:
Fails CI, trailing whitespace?

Reply:
Indeed, I wish this didn't display in such a confusing way :/
![image](https://user-images.githubusercontent.com/2223172/172375942-ca30b1fc-aa0c-4e36-ab89-05a249d885a9.png)